CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 cmake_minimum_required(VERSION 3.2)
+set(CMAKE_CXX_COMPILER "/usr/bin/clang++")
```

Comment:
This is not the correct position to set the path.
Do it below project below set(CMAKE_CXX_EXTENSIONS OFF)

Reply:
We can't do it below that line. It has to be declared [before the project variables are set](https://gitlab.kitware.com/cmake/community/-/wikis/FAQ#how-do-i-use-a-different-compiler). This method is marked to be avoided. The better method is to specify using the CLI option -D. If that's OK, I'll update the README instructions on how to compile via Clang by passing the language variable via the -D option. I will remove this line from the CMakeLists.txt.